### ISSUE TITLE
Error training ner using pretrain model id_ud-tag-dep-ner-1.0.0

### ISSUE THREAD

**Issue by Elvira-nlp** [danielgo13/spacyndo#2]
Hi Danielgo,

I try to training ner using pre-train model 'id_ud-tag-dep-ner-1.0.0'

But I have an error when I train ner using modify  train_ner.py

Traceback (most recent call last):
  File "train_ner_coba.py", line 155, in <module>
    main("id_ud-tag-dep-ner-1.0.0/","Final/",1000)
  File "train_ner_coba.py", line 91, in main
    nlp.update(
  File "/usr/local/lib/python3.8/site-packages/spacy/language.py", line 519, in update
    proc.update(docs, golds, sgd=get_grads, losses=losses, **kwargs)
  File "nn_parser.pyx", line 457, in spacy.syntax.nn_parser.Parser.update
  File "_parser_model.pyx", line 347, in spacy.syntax._parser_model.ParserStepModel.begin_update.backprop_parser_step
ValueError: operands could not be broadcast together with shapes (1,74) (73,) (1,74)

Could you help me? What should I do?

Thanks

**Comment by danieldanuega** [comment]
Hi @Elvira-nlp, did you use the model from this path? spacyndo/Models/Spacy 2.1.x/NER/id_ud-tag-dep-ner-1.0.0/

**Comment by Elvira-nlp** [comment]
Sorry for my late replay @danielgo13 . Yes, I did use the model from spacyndo/Models/Spacy 2.1.x/NER/id_ud-tag-dep-ner-1.0.0/, and I also use the latest version of spaCy.

I did experiment training from your NER data.  I got model Spacy NER and I can use this model for the pre-train model for the next training. I just thought maybe the problem because of the different versions of spaCy. Is it true?
